### ISSUE TITLE
File size limit for CSV uploads should match in frontend and backend

### ISSUE THREAD

**Issue by nathancrtr** [CDCgov/prime-simplereport#4242]
## Description

A CSV upload user reported an `Unexpected end of JSON input` error upon attempting an upload. Investigation discovered that the Spring service was rejecting the file as too large:

```
org.apache.tomcat.util.http.fileupload.impl.FileSizeLimitExceededException: The field 1 exceeds its maximum permitted size of 1048576 bytes.
```

This is due to the much higher file size limit enforced by the client (50MB vs 10MB) - the local validation passed, allowing the submission attempt, before being rejected with a confusing error message.

## Acceptance criteria
* The file size limits enforced by the client and server should be identical
    - Can bring the FE limit down, or the BE limit up
    - All else being equal, I would propose the latter

[Slack convo](https://skylight-hq.slack.com/archives/C02HKNHGCP6/p1661898385159849)

**Comment by mehansen** [comment]
1048576 bytes = 1 MB, right? so this is even more drastically lower than the 50 MB limit from the frontend

**Comment by nathancrtr** [comment]
Closed by #4300 ([link](https://github.com/CDCgov/prime-simplereport/pull/4300#discussion_r977990544)).
